### ISSUE TITLE
warning: incorrectly formatted message received (b[0] > 0xf4)

### ISSUE THREAD

**Issue by radix42** [whyrusleeping/zmsg#1]
I can't tell if this is an actual problem or not....is this what you see for normal sends of ZEC to your zaddr that don't have a message? Because I see 4 of the above and nothing else when I run "zmsg check"

**Comment by whyrusleeping** [comment]
@radix42 Yeah, Thats currently what you'll see. I'm not sure the best way to expose that.
We could just flat out ignore those messages unless theres a verbose flag enabled.

**Comment by whyrusleeping** [comment]
@radix42 It also looks like youre running an older version of the code, try pulling latest master (or `go get -u github.com/whyrusleeping/zmsg` if using go get)

**Comment by whyrusleeping** [comment]
It turns out i had forgotten to push a commit, try now. Should be a bit better.

**Comment by radix42** [comment]
I would ignore messages that don't have anything in the memo field at all

**Comment by whyrusleeping** [comment]
@radix42 Yeah, i'm doing that now. Weird thing though is that the memo field of a transaction sent without a memo specified has a single byte in it, `0xf6`. Not sure why. 

cc @zookozcash

**Comment by radix42** [comment]
that is weird, in light of https://github.com/zcash/zcash/issues/1849

I wonder how closely zcash v1.0.3 tracks that...or if that's just guidance for how to interpret what's in the memo field, and not enforced by code in any way

**Comment by zookozcash** [comment]
Hm, so with MagicBean-1.0.3, if the user doesn't provide a memo, then it puts `0xf6` followed by all `0x00` bytes? That's okay. That's roughly consistent with https://github.com/zcash/zcash/issues/1849#issuecomment-260169726 . I guess we should update the spec to say that `0xf60x000x00…` is how MagicBean signals "There is no memo here.".
